CONTRIBUTING.md

```diff
@@ -56,31 +56,31 @@ Some notes on GitHub PRs:
 ## Release Process (for Admin only)
 
 The release tags of this package follow exactly the MeiliSearch versions.<br>
-It means the `v0.17.0` tag in this repository corresponds to the DO image running with MeiliSerach `v0.17.0`.
+It means that, for example, the `v0.17.0` tag in this repository corresponds to the DO image running MeiliSerach `v0.17.0`.
 
 This repository currently does not provide any automated way to test and release the DO image.<br>
 **Please, follow carefully the steps in the next sections before any release.**
 
 ### Test before Releasing
 
-1. In (`scripts/deploy.sh`)[/scripts/deploy.sh] and [`server/tools/build-image.py`](/server/tools/build-image.py), change all the `vX.X.X` by the new MeiliSearch version. If you want to test with a MeiliSearch RC, replace them by the right RC version tag (`vX.X.XrcX`).
+1. In [`scripts/deploy.sh`](/scripts/deploy.sh) and [`server/tools/build-image.py`](/server/tools/build-image.py), change all the `vX.X.X` by the new MeiliSearch version. If you want to test with a MeiliSearch RC, replace them by the right RC version tag (`vX.X.XrcX`).
 
-2. Commit your changes on a new branch and open the related PR.
+2. Commit your changes on a new branch.
```

Comment:
For RC a PR isn't needed. For testing, a PR isn't needed. So I though it was better to move the PR part to when you actually want to publish the image in the next section. Let me know what you think @curquiza

Reply:
Hum! I'm not sure to understand. Let's talk about it directly when possible 🙂